src/Link.cc

```diff
@@ -578,12 +584,24 @@ bool Link::EnableWind() const
   return this->dataPtr->enableWind;
 }
 
+/////////////////////////////////////////////////
+bool Link::EnableGravity() const
+{
+  return this->dataPtr->enableGravity;
+}
+
 /////////////////////////////////////////////////
 void Link::SetEnableWind(const bool _enableWind)
 {
   this->dataPtr->enableWind = _enableWind;
 }
 
+/////////////////////////////////////////////////
+void Link::SetEnableGravity(const bool _enableGravity)
```

Comment:
```suggestion
void Link::SetEnableGravity(bool _enableGravity)
```
the header file doesn't use `const`, so remove it here for consistency. (I expect you copied from `SetEnableWind`, which should not have the `const` either. You may fix that as well if you like, but it is not required)

Reply:
Fixed `SetEnableWind()` as well.